mmdet/models/dense_heads/ld_head.py

```diff
@@ -0,0 +1,259 @@
+import torch
+from mmcv.runner import force_fp32
+
+from mmdet.core import (bbox2distance, bbox_overlaps, distance2bbox,
+                        multi_apply, reduce_mean)
+from ..builder import HEADS, build_loss
+from .gfl_head import GFLHead
+
+
+@HEADS.register_module()
+class LDHead(GFLHead):
+    """LD is the extension of knowledge distillation on localization task,
+    which utilizes the learned bbox distributions to transfer the localization
+    dark knowledge from teacher to student.
+
+    <https://arxiv.org/abs/2102.12252>
```

Comment:
Original paper: \`Localization Distillation for Object Detection \<https://arxiv.org/abs/2102.12252>`_